app/controllers/providers/about_financial_means_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Providers
+  class AboutFinancialMeansController < ProviderBaseController
+    def show
+      @name = full_name
+      @applicant = applicant
+    end
+
+    def update
+      continue_or_draft
+    end
+
+  private
+
+    def full_name
```

Comment:
There is already an `applicant.full_name` helper method on the model, you should be able to use that